tests/unit/s2n_tls_hybrid_prf_test.c

```diff
@@ -135,22 +98,30 @@ int main(int argc, char **argv)
         s2n_stuffer_write(&combined_stuffer, &classic_pms);
         s2n_stuffer_write(&combined_stuffer, &kem_pms);
 
-        EXPECT_SUCCESS(convert_hex_to_bytes(conn->secure.client_random, client_random_hex_in));
-        EXPECT_SUCCESS(convert_hex_to_bytes(conn->secure.server_random, server_random_hex_in));
-
-        EXPECT_SUCCESS(s2n_alloc(&conn->secure.client_key_exchange_message, (strlen(client_key_exchange_message_1) + strlen(client_key_exchange_message_2)) / 2));
-        const char *strings[] = {client_key_exchange_message_1, client_key_exchange_message_2};
-        EXPECT_SUCCESS(multi_convert_hex_to_bytes(conn->secure.client_key_exchange_message.data, strings, 2));
-
+        /* Prove to ourselves that client_random and server_random are the same length
+         * so that we can combine both assignments in a single loop. */
+        eq_check(CLIENT_RANDOM_LENGTH, SERVER_RANDOM_LENGTH);
+        for (uint32_t j = 0; j < CLIENT_RANDOM_LENGTH; j++) {
+            conn->secure.client_random[j] = client_random[j];
+            conn->secure.server_random[j] = server_random[j];
+        }
```

Comment:
I realize now why you didn't `s2n_blob`-ify this, instead I'd like to `memcpy_check`-ify these.

Reply:
Done